### DISCUSSION TITLE
Create - Create Another - Cancel

### DISCUSSION THREAD

**Discussion by AleksandarNeustadt** [filamentphp/filament#3147] [General]
Hi, 
I want to know, where i switch off/on option Create - Create Another - cancel.

I ask, because sometimes i have only create - cancel option when i open form (for example, for Category creation)

**Comment by ariaieboy** [comment]
On your CreateCategory.php page, you can switch these Actions off.

for switching off the Create Another you can set `protected static bool $canCreateAnother = false;`

if you want to disable all these Actions or replace them with your custom Actions you can override `protected function getFormActions(): array` in your Create page file
